src/containers/Accounts/AccountHeader/index.tsx

```diff
@@ -21,16 +21,63 @@ const CURRENCY_OPTIONS = {
   maximumFractionDigits: 6,
 }
 
-const AccountHeader = (props) => {
+interface AccountHeaderProps {
+  onSetCurrencySelected: Function
+  currencySelected: string
+  loading: boolean
+  accountId: string
+  data: {
+    balances: {
+      XRP: number
+    }
+    paychannels: {
+      // eslint-disable-next-line camelcase
+      total_available: string
+      channels: any[]
```

Comment:
Why any

Reply:
The previous definition was `PropTypes.shape({ length: PropTypes.number })`.  No other properties are referenced so I figured any[] array matches that.  This might be something to address when following up on #587.